.github/CODEOWNERS

```diff
@@ -0,0 +1,3 @@
+/README.md @fastly/developer-relations
+/fastly.toml @fastly/developer-relations
+*.js @fastly/ecp-sdk-sme-javascript
```

Comment:
Hmm. So GitHub is complaining that this team either doesn't exist or should have write access to the repo.

I found the team, so they do exist, and I've added them to this repo with write access BUT there are _zero_ members of that team so no one is going to get notified 😬

Reply:
Ok, this team is a codeowner on other starter kits, so I will raise it in the appropriate slack channel.

Reply:
Removing for now